tsl/src/continuous_aggs/materialize.c

```diff
@@ -163,23 +171,32 @@ continuous_agg_materialize(int32 materialization_id, bool verbose)
 				 NameStr(cagg_data.user_view_name));
 	}
 
+	caggs = ts_continuous_aggs_find_by_raw_table_id(cagg_data.raw_hypertable_id);
+	drain_invalidation_log(cagg_data.raw_hypertable_id, &invalidations);
+	catalog = ts_catalog_get();
+	materialization_invalidation_log_table_relation =
```

Comment:
I'd prefer that the open and close be put in `insert_materialization_invalidation_logs` to be consistent with the other functions in this file, and to not clutter up the top-level function, but it's not a strong objection.

Reply:
I agree here, but also not strongly.